csharp/OnnxRuntime.CSharp.proj

```diff
@@ -8,10 +8,19 @@ CMake creates a target to this project
 <Project DefaultTargets="Build">
   
   <PropertyGroup>
+    <OrtPackageId Condition=" '$(OrtPackageId)' == '' ">Microsoft.ML.OnnxRuntime</OrtPackageId>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <OutputPath>bin\$(Platform)\$(Configuration)\</OutputPath>
     <DotNetExe Condition= " '$(DotNetExe)' == '' ">dotnet</DotNetExe>
+    <NugetExe Condition= " '$(NugetExe)' == '' ">nuget</NugetExe>
```

Comment:
we use nuget.exe along with a generated nuspec to generate the nuget with the native shared library artifacts (dlls, so, ...)